onnx/defs/math/defs.cc

```diff
@@ -1070,7 +1008,21 @@ ONNX_OPERATOR_SET_SCHEMA(
 
               schema.BuildFunction(functionProto);
               return true;
-            }));
+            })
+        .SetContextDependentFunctionBodyBuilder(
+            [](const FunctionBodyBuildContext& ctx, const OpSchema& schema, FunctionProto& functionProto) -> bool {
```

Comment:
Nit: May be adding a comment "Function for opset 18" would be helpful. I realize the number "18" is down below, but it took me some time to realize it.